builtin/fstk/ui.lua

```diff
@@ -54,29 +54,42 @@ end
 --------------------------------------------------------------------------------
 --------------------------------------------------------------------------------
 
+local function wordwrap_hack(str)
```

Comment:
This is definitely a hack. It doesn't wrap the words but after 80 chars. It doesn't matter whether it is in the middle of a word or not.